receiver/simpleprometheusreceiver/factory.go

```diff
@@ -16,12 +16,14 @@ package simpleprometheusreceiver // import "github.com/open-telemetry/openteleme
 
 import (
 	"context"
+	"sync"
 	"time"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/config/confignet"
 	"go.opentelemetry.io/collector/consumer"
+	"go.uber.org/zap"
```

Comment:
Needs an update to `go.mod` to be able to use this

Reply:
Done